tsconfig.json

```diff
@@ -35,5 +35,11 @@
       "@/*": ["src/*"]
     }
   },
-  "include": ["**/*.ts", "**/*.tsx", "**/*.cjs", "**/*.astro"]
+  "include": ["**/*.ts", "**/*.tsx", "**/*.cjs", "**/*.astro"],
+  "ts-node": {
```

Comment:
Sybciej było mi to dorzucić, niż opisywać. Mamy teraz jeden tsconfig i nie musimy dodawać flag przy wywoływaniu `ts-node`